mettle/src/main.c

```diff
@@ -95,6 +127,10 @@ void parse_default_args(struct mettle *m)
 		"                                                               "
 		"                                                               "
 		"                                                               "
+		"                                                               "
```

Comment:
We should update [mettle.rb:47](https://github.com/bcook-r7/mettle/blob/f928937ab063d35f15d4aaa8e570b123e828cf29/lib/metasploit_payloads/mettle.rb#L47) and [mettle.rb:71](https://github.com/bcook-r7/mettle/blob/f928937ab063d35f15d4aaa8e570b123e828cf29/lib/metasploit_payloads/mettle.rb#L71) to account for this extra space.